docs/docs/choose-provider/index.md

```diff
@@ -0,0 +1,135 @@
+* **Identity and Access Management (IAM)**
+Identity and Access Management, or IAM, allows you to create users with limited or controlled access to your AWS services.
+> Ensure that your IAM user has permissions/access to all services required to build your stack! This often includes Route53, and EIP!
+ 
+ * ***[AWS Documentation](https://aws.amazon.com/iam/)***
+ * ***[Using IAM in your Stack](/docs/setup-aws-iam-user)***
+ * ***[Stack Reference](/docs/aws_eip)***
+
+* **Amazon Machine Images (AMI)**
+Every EC2 instance is built using an image. This image specifies the operating system and software that is created with the instance. AWS allows you to use any of their default AMIs or to create your own. 
+  * ***[AWS Documentation](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)***
+  * ***[Using AMIs in your Stack](/docs/using-amis)***
+
+* **Virtual Private Cloud (VPC)**
+AWS VPC is a Virtual Network! It allows you to create an complete logical network infrastructure for your VMs. This opens up a world of possibilities for using multiple VMs together. If you don't require any special networking, Koding handles this automatically. However, if you want to create a custom network environment, you can configure it simply within your Stack.
+
+ * ***[AWS Documentation](https://aws.amazon.com/vpc/)***
+ * ***[Using IAM in your Stack](/docs/create-an-aws-vpcr)***
+ * ***[Stack Reference](/docs/awc_vpc)***
+
+***
+
+## Vagrant <a name="vagrant"></a>
+
+Vagrant allows you to use your own machine to host your Koding VMs. You will install it along-side VirtualBox, so Vagrant can interface you with all of its features. 
+
+***[Creating a Stack with Vagrant](/docs/vagrant/creating-a-stack-with-vagrant)***
+***[Vagrant Website](https://www.vagrantup.com/)***
+***[Vagrant Stack Reference](/docs/vagrant/vagrant-stack-reference)***
+
+> Since (VirtualBox-based) Vagrant VMs do not exist in the Cloud, backing up your data is all the more important. If the machine that hosts your VMs dies, you'll lose those VMs and everything on them. Your Koding Stack will not be affected, so getting started with a new VM on a new machine will be quick and simple! 
+
+***
+
+## DigitalOcean <a name="digital-ocean"></a>
+
+***[Getting Started with DigitalOcean](/docs/getting-started-digital-ocean)***
+***[DigitalOcean Website](https://www.digitalocean.com/)***
+***[DigitalOcean Stack Reference](https://www.terraform.io/docs/providers/do/index.html)***
+
+DigitalOcean is a very popular option among developers an teams who don't need all of the features of AWS or GCE and also don't need all of the headache. Its feature-set is much smaller than the others, but those it does offer are the features most used by small teams. Documentation is limited (though this is countered by a strong active community), but since they are such a straightforward service, there isn't much required. DigitalOcean's pricing is among the best, in large-part due to their bare-bones approach. 
```

Comment:
"an" -> "and".  Remove the line about headache.  "smaller" -> "simpler".  Remove the word strong in "strong active community".